marketing/index.html

```
@@ -0,0 +1,229 @@
+        <div>
+          <h2>Build mods with confidence</h2>
+          <p>
+            The API includes side-aware base types, lifecycle callbacks, custom messaging, and typed TOML utilities for
+            addon configuration. Build companion client mods, server-only behavior, or both.
+          </p>
+          <a class="button button-primary" href="../docs/modding/overview.md">Explore Modding Guides</a>
+        </div>
+        <pre class="code-block"><code>// Example intent (conceptual)
+public sealed class EconomyTweaks : ServerModBase
+{
+    public override string Name => "EconomyTweaks";
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)